doc/requirements.txt

```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
I am not sure why the downgrade to 3.8, but I guess it doesn’t break anything either.

Reply:
Yes, I was also annoyed by it. The issue is: the machine I was doing it on has only 3.8, no way around it. The files is generated by Python and puts its version there. I could manually edit it back to 3.10 but that would be a lie. I thought that I rather avoid lying here at the expense of a bigger diff. Functionally, there's probably no difference...

Reply:
All right, god bless Docker, I did it with 3.10...